gradle/libs.versions.toml

```diff
@@ -15,9 +15,9 @@ androidx-test-junit = "1.1.2"
 androidx-test-orchestrator = "1.3.0"
 androidx-test-runner = "1.3.0"
 androidx-workmanager = "2.7.1"
-aws-kotlin = "0.29.1-beta" # ensure proper aws-smithy version also set
```

Comment:
Kotlin SDK updated coroutines to 1.7.3. Should we do the same ? (We are currently at 1.6.3.
Also updated are Kotlin to 1.9.10 (current version in amplify is "1.7.10").
Please confirm the breaking change in v0.31.0-beta does not impact us: **prefix generated endpoint and auth scheme providers with client name and track upstream changes**
Please verify **BREAKING: Refactor HttpCall and HttpResponse types**  in v.32.0-beta does not impact us
Please verify https://github.com/awslabs/aws-sdk-kotlin/issues/946 **Refactor CredentialsProvider APIs** does not impact us